Assets/Scripts/Fire.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+
+public class Fire : MonoBehaviour {
+    public float scaleFire;
+    public float speedDreasing;
+    public float minScale;
+    public int densityFire;
+    private ParticleSystem part;
+    private ParticleSystem.ShapeModule sh;
+    private ParticleSystem.EmissionModule em;
+
+    void Start() {
+        part = this.GetComponent<ParticleSystem>();
```

Comment:
you can omit the 'this' in C#

```suggestion
        part = GetComponent<ParticleSystem>();
```